demos/graph-classification/dgcnn-graph-classification.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>This first cell will need to be split up into a cell with the title only and then a cell with the rest of the text. This is so the "Run" buttons are placed right after the title. This requirement is being noticed on CI https://buildkite.com/stellar/stellargraph-public/builds/3588#8165461b-3270-4d18-9fe3-e52abf74a0b2/181-204 , but it's not failing properly (https://github.com/stellargraph/stellargraph/issues/1409). (https://github.com/stellargraph/stellargraph/pull/1363 has more info on the specifics of the check.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>